exdb/templates/exdb/experience_view.html

```diff
@@ -5,11 +5,18 @@
     {{ block.super }}
     <link rel="stylesheet" type="text/css" href="{% static 'exdb/css/experience_approval.css' %}" />
 {% endblock %}
+
+{% block header_color %} {{ experience.get_status_display|cut:" " }}-background{% endblock %}
+{% block header_title %}
+    <h2>{% trans "View" %} {{ experience.name }}</h2>
+{% endblock %}
+
 {% block content %}
-    <p><a href="{% url 'home' %}">{% trans "Return Home" %}</a></p>
-    <h1>{{ experience.name }}</h1>
-    <hr/>
-    <div class="experience-display">
+    <div class="row">
+        <div class="small-12 small-centered columns">
+            <h1 class="center">{{ experience.name }}</h1>
```

Comment:
Foundation has a helper class "text-center" that does the same as this "center" class.